core/src/main/java/org/elasticsearch/search/sort/FieldSortBuilder.java

```diff
@@ -83,6 +88,8 @@ public FieldSortBuilder missing(Object missing) {
      */
     @Deprecated
     public FieldSortBuilder ignoreUnmapped(boolean ignoreUnmapped) {
+        deprecationLogger.deprecated("[ignoreUnmapped]  parameter will be removed"
+                + "in the next major version. Use unmapped_type instead.");
```

Comment:
I don't think we need to add logging to the builders: this only applies to users of the java API that would already get compiler warnings because of the deprecated annotation